vms/volunteer/views.py

```diff
@@ -25,7 +25,9 @@
 from shift.services import get_volunteer_report, calculate_total_report_hours
 from volunteer.forms import ReportForm, SearchVolunteerForm, VolunteerForm
 from volunteer.models import Volunteer
-from volunteer.services import delete_volunteer_resume, search_volunteers, get_volunteer_resume_file_url
+from volunteer.services import (delete_volunteer_resume, search_volunteers,
+                               get_volunteer_resume_file_url, has_resume_file,

```

Comment:
oopsie. Almost there. 
```
from volunteer.services import (delete_volunteer_resume, search_volunteers,
                                get_volunteer_resume_file_url, has_resume_file,
                                get_volunteer_by_id)

``` 
one space missing on line 29 and 30

Reply:
@tonythomas01 Done with this too. Please review.